master/internal/api_user_intg_test.go

```diff
@@ -992,48 +992,74 @@ func TestGetAllAccessTokens(t *testing.T) {
 
 	createTestToken(ctx, t, api, userID1)
 
-	tokenInfo1, err := api.GetAccessToken(ctx, &apiv1.GetAccessTokenRequest{
-		UserId: int32(userID1),
+	usernameForGivenUserID, err := getUsernameForGivenUserID(ctx, userID1)
+	require.NoError(t, err)
+	filter := fmt.Sprintf(`{"username":"%s"}`, usernameForGivenUserID)
+
+	tokenInfo1, err := api.GetAccessTokens(ctx, &apiv1.GetAccessTokensRequest{
+		Filter: filter,
 	})
 	require.NoError(t, err)
 	require.NotNil(t, tokenInfo1)
-	require.Equal(t, int32(userID1), tokenInfo1.TokenInfo.UserId)
+	// Loop through all the returned user sessions
```

Comment:
should this be user session or access token records?

Reply:
Yes, will correct this